firefox_puppeteer/ui/menu.py

```diff
@@ -21,7 +21,7 @@ def menus(self):
         """
         menus = (self.marionette.find_element(By.ID, 'main-menubar')
                                 .find_elements(By.TAG_NAME, 'menu'))
```

Comment:
You also want to get rid of the `main-menubar` line here. You can retrieve the element via `self.element` now, which makes this class really general.